picard/ui/matchdetailsdialog/matchdetails.py

```diff
@@ -0,0 +1,211 @@
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+
+import os
+from pathlib import Path
+import traceback
+
+from PyQt5 import QtCore
+from PyQt5.QtWidgets import QFileDialog
+
+from picard import (
+    PICARD_VERSION_STR,
+    config,
+    log,
+    match_details_log,
+)
+from picard.const import (
+    ACOUSTID_HOST,
+    ACOUSTID_KEY,
+)
+from picard.track import Track
+
+from picard.ui.matchdetailsdialog import MatchDetailsDialogBase
+
+
+class MatchDetailsDialog(MatchDetailsDialogBase):
+
+    dialog_header_state = "matchdetailsdialog_header_state"
+    current_file = ""
+
+    options = [
+        config.Option("persist", dialog_header_state, QtCore.QByteArray())
+    ]
+
+    def __init__(self, parent):
+        """ Initialize the dialog box object with information about itself, including the columns in the table/grid."""
+        super().__init__(parent)
+        self.file_ = None
+        self.setWindowTitle(_("Match Details"))
+        self.columns = [
+            ('nbr',                  "_Log\nSeq"),
+            ('match_dtl_entry_type', "_Entry Type"),
+            ('is_new',               "_Better?"),
+            ('rec_id',               "_Recording ID"),
+            ('rec_title',            "Recording Title"),
+            ('rec_len',              "Recording\nLength"),
+            ('rec_artist',           "Recording\nArtist"),
+            ('rg_id',                "_Release Group ID"),
+            ('rg_title',             "Release Group\nTitle"),
+            ('rg_pri_type',          "Release Group\nPri Type"),
+            ('rg_sec_type',          "Release Group\nSec Types"),
+            ('rel_id',               "_Release ID"),
+            ('rel_ctry',             "Release\nCountry"),
+            ('rel_dt',               "Release\nDate"),
+            ('med_fmt',              "Medium\nFormat"),
+            ('med_pos',              "Medium\nPosition"),
+            ('med_tot_cnt',          "Medium\nCount"),
+            ('trk_id',               "_Track ID"),
+            ('trk_pos',              "Track\nPosition"),
+            ('med_trk_cnt',          "Medium\nTrack\nCount"),
+            ('rel_tot_trk_cnt',      "Release\nTotal Track\nCount"),
+            ('parts',                "_Scoring Parts"),
+            ('release_parts',        "_Scoring Release Parts"),
+            ('recording_base_score', "_Recording\nBase Score\n(Sources Ratio)"),
+            ('acoust_id',            "AcoustId ID"),
+            ('rec_src_cnt',          "AcoustId\nRecording\nSource\nCount"),
+            ('similarity',           "Similarity")
+        ]
+
+    def load_match_details(self, track, file_):
+        """ Retrieves data and loads it to the screen/dialog box """
+        if file_ is not None:
+            # NOTE: some of the values, like meta, are not exposed as constants
+            fp = file_.acoustid_fingerprint if hasattr(file_, 'acoustid_fingerprint') else ""
```

Comment:
You can use [`getattr()`](https://docs.python.org/3/library/functions.html#getattr) optional `default` parameter:

```python
fp = getattr(file_, 'acoustid_fingerprint', '')
```